static/js/muting_ui.js

```diff
@@ -158,6 +159,11 @@ export function toggle_topic_mute(message) {
     }
 }
 
+export function rerender_on_user_update() {
```

Comment:
Let's call this `rerender_for_muted_user`

Reply:
This was addressed in #18233 😃